### PR TITLE
refactor(components): add callbacks for change route

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
refactor(components): add callbacks for change route

### CODE REVIEW

File: src/components/DwellingResponse/index.js [hunk 1]
```diff
@@ -26,11 +30,13 @@ class DwellingResponse extends Component {
...
     componentDidMount() {
-        this.props.requestSurvey(this.props.id);
+        const {id} = this.props.match.params;
+        this.props.requestSurvey(id);
     }
 
     componentWillReceiveProps(nextProps) {
         const {survey} = nextProps;
+        this.state.survey = survey;
         if (survey && !survey.dwellingResponse) {
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]:
```javascript
componentWillReceiveProps(nextProps) {
  if (nextProps.survey) {
    // ...
  }
}
```

File: src/components/DwellingResponse/index.js [hunk 2]
```diff
@@ -43,6 +49,17 @@ class DwellingResponse extends Component {
         }));
     }
 
+    goToAddressList() {
+        const {address} = this.state.survey;
+        this.props.onPrev(address);
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]: rename `onPrevious`

File: src/util/index.js [hunk 3]
```diff
@@ -2,10 +2,14 @@ import canDrawQuestion from './canDrawQuestion';
 import cleanChildrenQuestions from './cleanChildrenQuestions';
 import getQuestions from './getQuestions';
 import getQuestionsWithParents from './getQuestionsWithParents';
+import historyPropTypes from './historyPropTypes';
 import questionPropTypes from './questionPropTypes';
+import matchPropTypes from './matchPropTypes';
```
Comment (navarroaxel) [inline src/util/index.js]: rename `matchParamsIdPropTypes`

**Comment by maximilianoforlenza** [comment]
listo

**Pull request merged**